README.md

```diff
@@ -43,15 +43,20 @@ We have few paths to lower down to the Torch MLIR Dialect.
 
 ## Install torch-mlir snapshot
 
-This installs a pre-built snapshot of torch-mlir for Python 3.7/3.8/3.9/3.10 on Linux and macOS.
+At the time of writing, we release pre-built snapshot of torch-mlir for Python 3.10 on Linux and macOS.
+Therefore, it is recommended to have Python 3.10 with conda.
```

Comment:
It doens't have to be conda. I would remove that or add it in addition to the "regular" Python users (who install 3.10 from Python.org)

Reply:
Done